cylc/flow/cfgspec/suite.py

```diff
@@ -85,6 +85,35 @@
             "suite-priority".
         ''')
 
+    with Conf('scheduler'):
+        Conf('includes', VDR.V_STRING_LIST, desc='''
+            Configure the directories and files to be included in the remote
+            file installation.
+                .. note::
```

Comment:
Had a go at building, this RST is creating:

* bullet point list (desired)
* inside a definition list (unwanted)
* inside a note (desired)
* inside a blockquote (unwanted)

Spot the slight indentation and the funny grey box in this screenshot:

<img width="650" alt="Screenshot 2020-09-14 at 14 31 24" src="https://user-images.githubusercontent.com/16705946/93092147-fdb13d80-f696-11ea-96aa-9b46a035ef32.png">

This is because RST is really sensitive to new lines and indentation. RST uses indentation like Python and it uses newlines to escape from directives.

if you change this:

```rst
             Configure the directories and files to be included in the remote
             file installation.
                 .. note::
                    These, as standard, include the following directories:
                         * app
                         * bin
                         * etc
                         * lib
                     And include the server.key file (from the .service
                     directory), this is required for authentication.
```

To this it should be fine:

```rst
             Configure the directories and files to be included in the remote
             file installation.

             .. note::
                These, as standard, include the following directories:

                 * app
                 * bin
                 * etc
                 * lib

                 And include the server.key file (from the .service
                 directory), this is required for authentication.
```

Reply:
Wow, yes, now I spot it. Very subtle, thanks for the help!